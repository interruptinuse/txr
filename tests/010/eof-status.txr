@(next (open-command "echo a; exit 5"))
@a
@(eof status)
