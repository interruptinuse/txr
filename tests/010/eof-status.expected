a="a"
status="5"
